### DISCUSSION TITLE
How should I set up the TouchScreen to use TFT70 with Raspberry Pi with klipperscreen ?

### DISCUSSION THREAD

**Discussion by teriyaki90** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2626] [Q&A]
I want to use klipperscreen in BIGTREETECH TFT70
But I can not use TouchScreen.
How should I set up ?

I was able to display the screen by adding the following to configtxt, but the touch screen cannot be used.

# uncomment to force a specific HDMI mode (this will force VGA)
hdmi_cvt:0=1026 600 60
hdmi_group:0=2
hdmi_mode:0=87

**Comment by padillla** [comment]
Hi @teriyaki90 
I am in a similar situation, but I have no idea how to plug and set the raspberry. 
Can you give me a hand? did you manage to enable the touch on it?

**Comment by radek8** [comment]
The firmware of this display is optimized to work with Marlin printer FW. For other systems, some functions may be limited.
The display has two modes:
touch - has its own graphic interface and communicates with the printer via a serial port, and is controlled by touch.
emulated - communicates via connector EXP1 and EXP2 (emulates LCD 12864 with ST7920), controlled by an encoder.
